plugins/auth-backend/src/providers/types.ts

```diff
@@ -203,34 +231,35 @@ export type AuthInfoBase = {
   scope: string;
 };
 
-export type AuthInfoWithProfile = AuthInfoBase & {
-  /**
-   * Profile information of the signed in user.
-   */
-  profile: ProfileInfo | undefined;
-};
-
-export type AuthInfoPrivate = {
+export type OAuthPrivateInfo = {
   /**
    * A refresh token issued for the signed in user.
    */
   refreshToken: string;
 };
 
+// {type: 'authorization_response', response: {...}}
+
 /**
  * Payload sent as a post message after the auth request is complete.
  * If successful then has a valid payload with Auth information else contains an error.
  */
-export type AuthResponse =
+export type WebMessageResponse =
   | {
-      type: 'auth-result';
-      payload: AuthInfoBase | AuthInfoWithProfile;
+      type: 'authorization_response';
```

Comment:
Déjà vu

Reply:
We tried only supplying the response or error to the function that sends the response, but that got more verbose than just listing the type a few more times.